jobrunner/log_utils.py

```diff
@@ -26,20 +26,23 @@ def formatting_filter(record):
     record.status = ""
 
     tags = {}
-
+    ctx = set_log_context.current_context
+    job = getattr(record, "job", None) or ctx.get("job")
+    req = getattr(record, "job_request", None) or ctx.get("job_request")
+    
     if hasattr(record, "status_code"):
         record.status = record.status_code
```

Comment:
I don't quite understand the renaming here

Reply:
So, the thinking was .status is always available, .status_code may be.

It might be too confusing, though.